tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+include::exercises.adoc[tag=exercise-2-5]
+
+Let’s look at inserting images next.
+>> Start next lesson
+
+==== Images
+
+There are two AsciiDoc image macro types, block and inline. 
+
+*Block image*
+
+A block image is displayed as a discrete element, i.e., on its own line, in a document. A block image is designated by image macro name and followed by two colons (`::`) It’s preceded by a blank line, entered on a line by itself, and then followed by a blank line.
+
+However, in standard documents images don’t just decorate text, but rather show important features. You might want to reference that image in the text, which you need an anchor for. You can also explain a graphic by adding a key. The entries of the key follow the syntax of a definition list (`key::Explanation`).
+
+[source, AsciiDoc]
+----
+[[figureC-1]] <1>
+.Typical gelatinization curve <2>
+image::images/rice_image2.png[Image of the gelatinization curve] <3>
+
+*Key* <4>
+
+stem:[t_90]:: time required to gelatinize 90 % of the kernels <5>
+P:: point of the curve corresponding to a cooking time of stem:[t_90]
+----
+<1> Anchor for references
+<2> Image title
+<3> Image macro `image::path/file.jpg[alt text]`. If you don’t want to write alt text, you still need to append empty square brackets at the end of the macro. 
+<4> Key beginning
+<5> Key entries
+
+*Inline image macro* 
+//Maybe only discuss in documentation but not in tutorial, as it won't be covered in the exercise?
+An inline image is displayed in the flow of another element, such as a paragraph. This is useful when you want to incorporate buttons or other icons into the text flow. 
+
+The inline image macro is almost identical to the block image macro, except its macro name is followed by a single colon (:). For more information refer to the https://docs.asciidoctor.org/asciidoc/latest/macros/images/[AsciiDoc documentation]. 
+
+
+[[exercise-2-6]]
+include::exercises.adoc[tag=exercise-2-6]
+
+Let’s look at admonitions next.
+>> Start next lesson
+
+==== Admonitions
+Admonitions are signal words used to catch the reader’s attention, such as “TIP”, “NOTE”, or “WARNING”. There are two ways to declare an admonition: inline and as a block.
+
+*Inline Admonitions*
+
+Start a new line with the signal word in all caps and a colon and write your admonition.
+
+.Example for an inline note
+[source, AsciiDoc]
+----
+NOTE: Advice on when to use which signal word is specified in ANSI Z535.6. 
+----
+
+*Block admonitions*
+
+. Start with the signal word in all caps enclosed in square brackets.
+. Insert the block delimiter.
+. Insert any AsciiDoc markup that you need.
+. End the block with a block delimiter.
+
+.Example for a note block.
+[source, AsciiDoc]
+------
+[NOTE] <1>
+==== <2>
+This is an example of an admonition block. <3>
+
+Unlike an admonition paragraph, it may contain any AsciiDoc content.
+The style can be any one of the admonition labels:
+
+* NOTE
+* TIP
+* WARNING
+* CAUTION
+* IMPORTANT
+==== <4>
+------
+
```

Comment:
```suggestion

Metanorma adds two more signal words: "Safety precautions" and "Danger". Since they are not standard  AsciiDoc functionality, you'll need to mark them with a `type` attribute like this: 

[source, AsciiDoc]
====
[type=danger]
DANGER: Do perform maintenance tasks while the machine is still operating. 
====

```